doc/faq.md

```diff
@@ -0,0 +1,24 @@
+- [What if I have some commands that are missing?](#what_if_i_have_some_commands_that_are_missing)
+
+
+## What is the license model of Dragonfly? Is it an open source?
+Dragonfly is released under [BSL 1.1](../LICENSE.md) (Business Source License). The BSL license is more permissive than AGPL-like licenses in some elements. In general terms, it means that dragonfly's code is free to use and free to change as long as you do not provide paying support or managed services for an in-memory datastore.
+We followed the trend of other technological companies like Elastic, Redis, MongoDB, Cockroach labs, Redpanda Data to protect our rights to provide service and support for the software we are building. 
+
+## Can I use dragonfly in production?
+License wise you are free to use dragonfly in your production as long as you do not provide dragonfly as a managed service. From a code maturity point of view, Dragonfly's code is tested with unit testing and also against the Redis testing TCL scripts. However as with any new software there are use cases that are hard to test and predict. We advise you to run your own particular use case on dragonfly for a few days before considering production usage. 
+
+## Dragonfly provides vertical scale, but we can achieve similar throughput with X nodes in a Redis cluster.
+Dragonfly utilizes the underlying hardware in an optimal way. Meaning it can run on small 8GB instances and scale verticly to large 768GB machines with 64 cores. This versatility allows to drastically reduce complexity of running cluster workloads to a single node saving hardware resources and costs. More importantly, it reduces the complexity (total cost of ownership) of handling the multi-node cluster. In addition, Redis cluster-mode imposes some limitations on multi-key and transactinal operations while Dragonfly provides the same semantics as single node Redis.
+
+## I get it, but why not change Redis open source?
+Many of the pain points we addressed like Multi-core support, memory optimization, forkless snapshot, improved eviction algo, unit testing and more... have been the achilles heel and a long standing requests by the community. Some were not addressed for as long as ten years, others were dismissed because they require a complete rewrite of the engine. We believe that innovation at this scale is impossible on Redis OSS. KeyDB reused Redis OSS to the limit and introduce multi-threaded architecture into it. But to do all that with minimal locking we had to think about all things holistically. How you handle backpressure? how you do snapshotting? How you implement multi-key operations or blocking transactions? For that you need  need special algorithms to provide atomicity, you need fibers/coroutines to be able to block your calling context yet unblock the cpu for other tasks etc. All this was designed bottom up from scratch. BTW, Dragonfly does use some Redis code around single value data-structures like zset, hset etc...
```

Comment:
Can you pls just leave the title? I think the answer is too verbose . I will add the answer some time this week.

Reply:
Its shorter now. If you still think it should not get in I will delete.

Reply:
I also do not wish to grill Redis in our repo. yeah, pls delete.